packages/bigtable/src/table.js

```diff
@@ -568,41 +556,48 @@ Table.prototype.getRows = function(options, callback) {
     objectMode: true
   };
 
-  if (options.key) {
-    reqOpts.rowKey = Mutation.convertToBytes(options.key);
-  } else if (options.start || options.end) {
-    reqOpts.rowRange = Table.formatRowRange_(options);
-  } else if (options.keys || options.ranges) {
-    reqOpts.rowSet = {};
+  if (options.start || options.end) {
+    options.ranges.push({
+      start: options.start,
+      end: options.end
```

Comment:
Yep, per proto docs there seem to be defaults when either start or end is absent.

```
// Specifies a contiguous range of rows.
message RowRange {
  // The row key at which to start the range.
  // If neither field is set, interpreted as the empty string, inclusive.
  oneof start_key {
    // Used when giving an inclusive lower bound for the range.
    bytes start_key_closed = 1;

    // Used when giving an exclusive lower bound for the range.
    bytes start_key_open = 2;
  }

  // The row key at which to end the range.
  // If neither field is set, interpreted as the infinite row key, exclusive.
  oneof end_key {
    // Used when giving an inclusive upper bound for the range.
    bytes end_key_open = 3;

    // Used when giving an exclusive upper bound for the range.
    bytes end_key_closed = 4;
  }
}
```